modules/swagger-codegen/src/main/resources/JavaPlayFramework/build.mustache

```diff
@@ -6,11 +6,5 @@ lazy val root = (project in file(".")).enablePlugins(PlayJava)
 
 scalaVersion := "2.11.7"
 
-libraryDependencies ++= Seq(
-javaJdbc,
-cache,
-javaWs{{#useSwaggerUI}},
-"org.webjars" % "swagger-ui" % "2.2.10-1"{{/useSwaggerUI}}{{#useBeanValidation}},
-"javax.validation" % "validation-api" % "1.1.0.Final"
-{{/useBeanValidation}}
-)
+libraryDependencies += {{#useSwaggerUI}}"org.webjars" % "swagger-ui" % "2.2.10-1"{{/useSwaggerUI}}
```

Comment:
Gah, you are right. i'll fix the code.

With regard to SBT best practices and `+=` vs `++=`, I've never seen any recommendation on one vs the other. It is not a problem to change it, though. One SBT best practice this project isn't following is using a Dependencies.scala file to declare dependencies ([source](http://jsuereth.com/scala/2013/06/11/effective-sbt.html), [example](https://github.com/playframework/playframework/blob/master/framework/project/Dependencies.scala)). Just mentioning that in case you want me to change that too.